src/bsoncxx/types.hpp

```diff
@@ -17,6 +17,7 @@
 #include <bsoncxx/config/prelude.hpp>
 
 #include <cstring>
+#include <chrono>
```

Comment:
This is a bit of a nitpick, but for each header subsection we prefer to sort them - so <chrono> goes before <cstring>